docs/integrations/asgi.md

```diff
@@ -34,15 +31,17 @@ async def app(scope, receive, send):
     )
     await send({"type": "http.response.body", "body": b"Hello, world!"})
 
-app = OpenTelemetryMiddleware(app)
+app = logfire.instrument_asgi(app)
 
 if __name__ == "__main__":
     import uvicorn
 
     uvicorn.run(app)
 ```
 
-You can read more about the OpenTelemetry ASGI middleware [here][opentelemetry-asgi].
+The keyword arguments of [`logfire.instrument_asgi()`][logfire.Logfire.instrument_asgi] are passed to the
+[`OpenTelemetryMiddleware`][opentelemetry.instrumentation.asgi.OpenTelemetryMiddleware] class
+of the OpenTelemetry ASGI Instrumentation package.
 
 ## Excluding URLs from instrumentation
 <!-- note that this section is duplicated for different frameworks but with slightly different links -->
```

Comment:
The note here should refer to instrument_asgi, not OpenTelemetryMiddleware.

In fact we should probably just implement the env var ourselves.